src/AltinnCore/Common/Services/Implementation/RepositorySI.cs

```diff
@@ -1205,31 +1205,31 @@ public bool DeleteService(string org, string app)
         }
 
         /// <summary>
-        /// Returns a list of all apps for a given organisation present in the local repository
+        /// Returns a list of all apps for a given org present in the local repository.
```

Comment:
shouldnt this be organisation?

Reply:
Have used org since that variable is explained to more detail in the parameter list.